src/main/java/picard/TestStringUtilSplit.java

```diff
@@ -38,6 +42,8 @@ public void run() {
         }
         watch.stop();
         System.out.println("StringTokenizer took " + watch.getElapsedTime());
+        tokenizerTime = watch.getElapsedTime();
+        return (tokenizerTime + 0D) / stringUtilTime;
     }
 
     public int split(final String s, final String[] tokens, final String token) {
```

Comment:
`new StringTokenizer(s, token, false)` is the same as `new StringTokenizer(s, token)`

Reply:
(not my code...) TIL